yt_dlp/extractor/playsuisse.py

```diff
@@ -150,6 +174,9 @@ def _get_media_data(self, media_id):
         return response['data']['assetV2']
 
     def _real_extract(self, url):
+        if not self._ID_TOKEN:
+            self.raise_login_required()
```

Comment:
```suggestion
            self.raise_login_required(method='password')
```
or else the error message will tell the user they can pass cookies too